### PR TITLE
Add Simplified Chinese localization

### PR DESCRIPTION
**Pull request by L1cardo**

### CODE REVIEW

**Comment by freshollie** [comment]
谢谢你! 

Looks like you’ve included some extra “,” at the end of some files. JSON doesn’t like this hence failing to compile. It will also be worth running “yarn extract-locales” locally as this will put the translations in the correct order in the files.

**Comment by L1cardo** [comment]
Oh！ Sorry for the extra “,” , I just realize that these are all JSON files, I will fix them right now

File: locales/zh/flashing.json [hunk 1]
```diff
@@ -1,106 +1,106 @@
...
+  "Could not load releases": "无法加载 releases",
+  "Could not load targets": "无法加载目标",
+  "Could not unlock device: {{message}}": "无法解锁设备: {{message}}",
+  "Could not use firmware": "无法使用固件",
+  "Could not write new firmware to radio": "无法写入新的固件到遥控器",
+  "Details": "详情",
+  "Device firmware may be read protected, preventing updates": "设备固件可能被读取保护，防止更新",
+  "Device successfully unlocked! You will need to re-enter DFU mode to continue flashing": "设备已成功解锁！您需要重新进入 DFU 模式来继续刷写",
+  "DFU connection active": "DFU 模式已激活",
+  "Done": "完成",
```
Comment (freshollie) [inline locales/zh/flashing.json:37]: To clarify, this is a button to confirm "I'm done"

It's the "Done" button in the bottom right after flashing has completed. Not done like finished I guess?

**Comment by freshollie** [comment]
Looks good apart from the `Done` translation. What is `[.output/buddy.box](https://github.com/EdgeTX/buddy/pull/23/files#diff-94ae4b18fb5cc1aba4e47ff988b6051430f3bac6cb10d4d5747387f702935c37)`?

File: locales/zh/flashing.json (hunk 1 above)
Comment (freshollie) [reply locales/zh/flashing.json:37]: This translation may be right, just wanted to clarify that it's to do with a button and not to notify that something is done.

Comment (L1cardo) [reply locales/zh/flashing.json:37]: ”完成” can mean "done" or "finished", that's makes you confused. It's correct though.

**Review by freshollie — CHANGES_REQUESTED** [review]
@L1cardo I will merge once `.output/buddy.box` is removed

**Comment by L1cardo** [comment]
> @L1cardo I will merge once `.output/buddy.box` is removed

Removed! Sorry for my carelessness. 😅

**Review by freshollie — APPROVED** [review]

**Pull request merged**